src/assessment.js

```diff
@@ -27,7 +27,7 @@ var q1 = /* your answer here */;
 //
 //     `['alderaan', 'deathStar', 'fire', 'global']`
 //
-var q2 = [/* your answer here */];
+var q2 = ['fire'];
```

Comment:
Something inside the function `fire` would have access to `laserQuantity`, but so would something inside `deathStar`.